### ISSUE TITLE
StackImageUpdateAvailable being marked as Resolved even if not yet updated image

### ISSUE THREAD

**Issue by kingp0dd** [mbecker20/komodo#240]
I've been looking for the "Image Update Available" icon in the dashboard and it Stacks list. I finally found it in "Alerts" page. 

However, I noticed that it automatically gets tagged as Resolved even if the image versions are not yet updated/redeployed.

Is this intended? I was hoping that I could see images that can be updated, since I disabled "Auto-Update".

**Comment by mbecker20** [comment]
Currently the update alerts are pretty barebones, and lack the "lifetime" feature of the system stat alerts. But no reason the lifetime feature cannot be added for the update alerts as you mention.

**Comment by bcat** [comment]
> I noticed that there's also a notification on the Stacks page. Would be nice if it could be shown on the Stacks List/Table page as well; i think that would serve the same purpose as the lifetime feature in the meantime - which is just to see available updates in one view

Agreed! IMO, it would be nice if this showed up in the dashboard too (N stacks running, M stacks out of date, etc.).
